litmus/PTX/Proxy-SingleThread-rf-surW-texR-genR.litmus

```diff
@@ -1,4 +1,4 @@
-PTX fence
+PTX Proxy-SingleThread-rf-surW-texR-genR
```

Comment:
Please explain that the wrong fence is being used

Reply:
Updated